### PR TITLE
[*WIP*] Implementing showcase design changes for jQuery

### PR DESCRIPTION
**Pull request by amarie401**
## Description
As apart of on-going efforts to separate jQuery from PF-core, I have been working on a showcase page to host all of pf-jQuery components #692 .

I have started the process of implementing the design templates/mockups for the showcase pages based off of these two pages: 

http://rawgit.com/junezhang/showcase/master/index.html
https://rawgit.com/patternfly/patternfly-react/gh-pages/index.html

Storybook does not currently work with jQuery. @junezhang could you please review and recommend any changes so far to what I have done? :)

## Link to rawgit and/or image
jquqery-showcase

### CODE REVIEW

File: tests/pages/jquery/index.hbs [hunk 1]
```diff
@@ -22,88 +22,77 @@
...
+        <div class="container-fluid">
+          <div class="navbar-header">
+            <button type="button" class="navbar-toggle">
+              <span class="sr-only">Toggle navigation</span>
+              <span class="icon-bar"></span>
+              <span class="icon-bar"></span>
+              <span class="icon-bar"></span>
+            </button>
+            <div class="navbar-brand">
+              <a id="logo" title="PatternFly Showcase" href="#">PATTERNFLY-JQUERY<span class="secondary-logo"> SHOWCASE</span></a>
```
Comment (junezhang) [inline tests/pages/jquery/index.hbs]: `<a id="logo" title="PatternFly jQuery" href="#">PATTERNFLY-<span class="secondary-logo">JQUERY</span></a>`
Here should be PatternFly-jQuery, replace the "SHOWCASE" by "JQUERY"

File: tests/pages/jquery/index.hbs [hunk 2]
```diff
@@ -22,88 +22,77 @@
...
+                <a href="#">Components</a>
+              </li>
+              <li>
+                <a href="#">Contribute</a>
+              </li>
+              <li>
+                <a href="https://github.com/patternfly/patternfly-jquery">Github</a>
+              </li>
+            </ul>
+          </div>
```
Comment (junezhang) [inline tests/pages/jquery/index.hbs]: Beside the github link, there is an icon for all the patternfly showcases. User can click that icon to switch to any one showcase.  
screen shot 2017-08-09 at 11 56 19 am

File: tests/pages/jquery/index.hbs [hunk 3]
```diff
@@ -22,88 +22,77 @@
...
+            <ul>
+              <li>
+                <a href="#">Components</a>
+              </li>
+              <li>
+                <a href="#">Contribute</a>
+              </li>
+              <li>
+                <a href="https://github.com/patternfly/patternfly-jquery">Github</a>
+              </li>
```
Comment (junezhang) [inline tests/pages/jquery/index.hbs:93]: Same with the above comment about the PatternFly showcase list icon.

File: tests/pages/jquery/index.hbs [hunk 4]
```diff
@@ -22,88 +22,77 @@
...
+            </ul>
+          </li>
+        </ul>
+      </nav>
+    </header>
 
     <div class="doc-wrapper">
       <div class="container">
-        <div id="doc-header" class="doc-header text-center">
-          <h1 class="doc-title">JQuery Components</h1>
```
Comment (junezhang) [inline tests/pages/jquery/index.hbs:87]: Is that "jQuery Components"?

**Review by junezhang — CHANGES_REQUESTED** [review]

**Comment by junezhang** [comment]
Thanks for your PR. @amarie401  I left some comments in the code lines.

**Comment by amarie401** [comment]
Thank you for your comments @junezhang. I will make the changes you have requested and follow up.

**Comment by amarie401** [comment]
@junezhang I have made the requested changes. Please let me know if there is anything missing or anything that should be added. Also I am having a small issue with the dropdown icon not aligning with the nav properly, it is a bit higher than the other li items although I am using the same html & css for the nav as in the rawgit link. Any suggestions on how to fix this? Thank you!

dropdown

**Comment by junezhang** [comment]
@amarie401 I'd like to help this issue. Could you give me an online link, I can debug.  Or you can tell me how to run your page, I have pulled your code, but don't know how to run it.

**Comment by amarie401** [comment]
@junezhang There was a directory error issue when trying to run the page that was discovered so I fixed that. Please pull down my new recent changes and then make sure you have node.js installed on your computer and do ```npm install``` in the console, then do ```npm run jquery-dev``` to run the script and it should work. The browser window should pop up and you should see something like ```http://localhost:3000/tests/jquery/``` in the address bar and see the page -- make sure you have the jquery-showcase remote branch pulled as well. Let me know if that works for you. I will work on trying to get a link. Right now we have it set up using a .hbs extension because we are using handlebars.js so I cannot provide a rawgit link currently until we finish configuring it to render as html.

**Comment by junezhang** [comment]
@amarie401 I fixed this issue on here: https://github.com/junezhang/patternfly/commit/482894e5a08d8540a59f84c98fb900d6b1ee7d21
I don't know how to add this commit to your PR. But you can copy this piece of css code to yours.

**Comment by amarie401** [comment]
@junezhang thank you! I will also begin work on the get started page to adapt to jQuery and would appreciate your feedback when I have completed it. I will add to this PR when I have completed it and let you know.

**Comment by amarie401** [comment]
@bleathem @junezhang I accidentally merged into the jquery-showcase branch on PF instead of my own jquery-showcase branch when I pushed my commit. I think it should be okay since I am currently the only one working in the jquery-showcase branch on PF but I will be more mindful going forward. I opened up a new PR here #712

**Comment by amarie401** [comment]
@junezhang I have added the "get-started" page. It is inside of tests/pages/jquery/src/get-started.html along with the get-started.css. Can you please review? Thank you. 

get-started

**Pull request merged**